src/screens/Login/Components/login-form.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Button, TextInput } from 'react-native-paper';
+import { View, Image } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+import { styles } from '../styles';
+import { Input, Text } from 'src/components';
+import { AppColor, AuthRoute } from 'src/common/enums';
+
+const LoginForm = () => {
```

Comment:
As long as this component is Form, let's keep only form related stuff here. And move logo and footer to the screen-component.